{"name": "test_post_user_invalid", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "def test_post_user_invalid():\n        outcome=api.post_user('https://reqres.in/api/users',data.post_user_invalid)\n        result={\"name\": \"\", \"id\": \"597\", \"createdAt\": \"2021-11-03T12:10:22.456Z\"}\n        if outcome==result:\n            assert True\n        else:\n>           assert False\nE           assert False\n\nPOM.py:145: AssertionError"}, "start": 1635950528920, "stop": 1635950529627, "uuid": "8e1a059e-fc49-4a27-985b-23d88a52d3a3", "historyId": "a2c2cd275d6a8da379bfb8146f40f307", "testCaseId": "01ea9d852da86b3943b0efe0b09084b6", "fullName": "Reqres_API.POM#test_post_user_invalid", "labels": [{"name": "parentSuite", "value": "Reqres_API"}, {"name": "suite", "value": "POM"}, {"name": "host", "value": "Foysal"}, {"name": "thread", "value": "11808-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Reqres_API.POM"}]}